src/pages/index.js

```diff
@@ -23,7 +26,25 @@ export default function Index({location}) {
             <SectionWarranty />
             <SectionFaq />
             <SectionContact />
+            <BlogSection fewPosts={fewPosts} />
             <Footer />
         </Layout>
     )
 }
+
+export const pageQuery = graphql`
+  query {
+    allWpPost(sort: { fields: [date] }) {
```

Comment:
To avoid querying unnecessary data we can limit the query to just the number of posts we intend to use. Also, the sorting step can be done here as well. Like that:

```suggestion
    allWpPost(sort: { fields: [date], order: DESC }, limit: 3) {
```

Reply:
Inserted that in cba5ad7! Thanks ❤️